src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -108,7 +242,13 @@ describe('PaypalCommerceButtonStrategy', () => {
             store,
             checkoutActionCreator,
             formPoster,
-            paypalCommercePaymentProcessor
+            paypalCommercePaymentProcessor,
+            orderActionCreator,
+            countryActionCreator,
+            new ConsignmentActionCreator(
+                new ConsignmentRequestSender(requestSender),
+                new CheckoutRequestSender(requestSender)
```

Comment:
You can use `consignmentActionCreator` here.

Reply:
fixed